packages/shared/src/features/onboarding/shared/PricingPlansV2.tsx

```diff
@@ -0,0 +1,233 @@
+        >
+          <Typography
+            bold
+            type={TypographyType.Footnote}
+            className="p-1 pt-0 text-center"
+          >
+            BEST VALUE
+          </Typography>
+          {children}
+        </div>
+      )}
+    >
+      <RadioItem
+        className={{
+          wrapper: classNames(
+            'flex flex-col gap-2 rounded-16 bg-white px-2 py-3',
+            !isBestValue && 'border',
+            isActive
+              ? 'border-action-share-default bg-brand-float'
+              : 'border-border-subtlest-secondary bg-surface-invert',
+          ),
+          content: 'flex flex-row items-center gap-2 font-normal',
+        }}
+        value={priceId}
+        checked={isActive}
+        onChange={() => onChange?.(priceId)}
+      >
+        {badge?.text && (
+          <Typography
+            type={TypographyType.Caption1}
+            tag={TypographyTag.Span}
+            className={classNames(
+              'absolute -top-3 left-3 inline-flex -translate-y-1/2 rounded-6 px-1 text-white',
+              isActive
+                ? badge.background ?? 'bg-brand-default'
+                : 'bg-surface-secondary',
+            )}
+            data-testid="plan-badge"
+          >
+            {badge.text}
+          </Typography>
+        )}
+        <div className="flex-1">
+          <Typography
+            bold
+            type={TypographyType.Body}
+            tag={TypographyTag.H4}
+            color={
+              isActive ? TypographyColor.Primary : TypographyColor.Tertiary
+            }
+          >
+            {label}
+          </Typography>
+          {!!price && (
+            <>
+              {oldPrice?.monthly && (
+                <Typography
+                  tag={TypographyTag.Del}
+                  className="mr-2"
+                  type={TypographyType.Footnote}
+                  color={TypographyColor.Quaternary}
+                >
+                  {oldPrice.monthly}
+                </Typography>
+              )}
+              <Typography
+                type={TypographyType.Footnote}
+                tag={TypographyTag.Span}
+                color={
+                  isActive ? TypographyColor.Primary : TypographyColor.Tertiary
+                }
+              >
+                {price.monthly.formatted}
+              </Typography>
+            </>
+          )}
+        </div>
+        {!!price && (
+          <div
+            className={classNames(
+              'flex items-end justify-end gap-5',
+              isActive ? 'text-text-primary' : 'text-text-tertiary',
+            )}
+          >
+            {oldPrice?.daily && (
+              <Typography
+                tag={TypographyTag.Del}
+                type={TypographyType.Footnote}
+                color={TypographyColor.Tertiary}
+              >
+                {oldPrice.daily}
+              </Typography>
+            )}
+            <div className="relative">
+              <svg
```

Comment:
Why is this a inline SVG and not a icon for example?

Reply:
This is not an actual icon, just this shape, didn't create a component because this will not be reused anywhere 👀 

<img width="43" alt="image" src="https://github.com/user-attachments/assets/f57066d4-c7e0-4c85-9887-b1a51b5461b9" />